test/functional/rpc_rawtransaction.py

```diff
@@ -432,27 +432,51 @@ def run_test(self):
 
         self.log.info('sendrawtransaction/testmempoolaccept with maxfeerate')
 
+        # Test a transaction with small fee
         txId = self.nodes[0].sendtoaddress(self.nodes[2].getnewaddress(), 1.0)
         rawTx = self.nodes[0].getrawtransaction(txId, True)
         vout = next(o for o in rawTx['vout'] if o['value'] == Decimal('1.00000000'))
 
         self.sync_all()
         inputs = [{ "txid" : txId, "vout" : vout['n'] }]
-        outputs = { self.nodes[0].getnewaddress() : Decimal("0.99999000") } # 1000 sat fee
+        outputs = { self.nodes[0].getnewaddress() : Decimal("0.999990000") } # 10000 sat fee
```

Comment:
eh, actually this is incorrect. Appending zeros after the number after the decimal point does not change the number.


```py
>>> 0.9 == 0.90
True

Reply:
Yeah, should be `0.99990000`

Reply:
OMG. I'll send a patch soon.

Reply:
Done in #16929.

Reply:
> Yeah, should be `0.99990000`

Done in #16929.